core/src/main/java/com/yahoo/ycsb/BasicDB.java

```diff
@@ -111,21 +124,23 @@ public Status read(String table, String key, Set<String> fields, HashMap<String,
 
 		if (verbose)
 		{
-			System.out.print("READ "+table+" "+key+" [ ");
+      StringBuilder sb = getStringBuilder();
```

Comment:
The main difference is minimising contention for System.out. We could create a new StringBuilder each time, and while not cheap it is small compared to the cost of writing to System.out